data/Nigeria/Senate/sources/instructions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/nigeria-national-assembly",
+        "query": "SELECT * FROM data WHERE js_position = 'Sen'"
+      },
+      "source": "http://www.nass.gov.ng/",
+      "type": "membership"
+    },
+    {
+      "file": "manual/terms.csv",
+      "type": "term"
+    },
+    {
+      "file": "wikidata/groups.json",
+      "type": "group",
+      "create": {
+        "from": "group-wikidata",
+        "source": "manual/group_wikidata.csv"
+      }
+    },
+    {
+      "file": "wikidata/elections.json",
+      "type": "wikidata-elections",
+      "create": {
+        "from": "election-wikidata",
+        "base": "Q22283753"
```

Comment:
I'm not sure that Q22283753 works as a base here. This includes _all_ general elections in Nigeria, even when those didn't include Senate elections. As far as I can tell, the Senate was only created in 1999, so we shouldn't really be importing elections from before then.